link-parser/link-parser.c

```diff
@@ -339,6 +339,26 @@ static void print_parse_statistics(Sentence sent, Parse_Options opts)
 	}
 }
 
+/**
+ * Check for the auto-next-linkage test request (for LG code development).
+ * It is given using the special command: test=auto-next-linkage[:display_max]
+ * when :display_max is an optional indication of the maximum number of
+ * linkages to auto-display (the default is DISPLAY_MAX).
+ * For example, to issue up to 20000 linkages for each batch sentence,
+ * the following can be used:
+ * link-parser -limit=30000 -test=auto-next-linkage:20000 < file.batch
+ */
+static int auto_next_linkage_test(const char *test)
+{
+	char auto_next_linkage_str[] = ",auto-next-linkage";
```

Comment:
maybe you are right, and here it doesn't make a much of a difference.  I just have a habit: use const as often as you can.  So that just jumped out at me.